src/components/LHNOptionsList/types.ts

```diff
@@ -45,7 +45,7 @@ type CustomLHNOptionsListProps = {
     contentContainerStyles?: StyleProp<ContentStyle>;
 
     /** Sections for the section list */
-    data: string[];
+    data: string[] | null;
```

Comment:
Are you sure `null` is needed? Try to follow other comments and I think you should be able to get rid of it